File: app/src/main/java/com/example/androidclient/RetrofitClient.java
package com.example.androidclient;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitClient {
    Retrofit retrofit = new Retrofit.Builder()
            .baseUrl("http://172.30.1.48:8787/")
            .addConverterFactory(GsonConverterFactory.create())
            .build();

    public ApiService apiService = retrofit.create(ApiService.class);
}

File: app/src/main/java/com/example/androidclient/retrofit/RetrofitClient.kt
package com.example.androidclient.retrofit

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object RetrofitClient {
    private var instance: Retrofit? = Retrofit.Builder()
        .baseUrl("https://narsha2020.herokuapp.com")
        .addConverterFactory(GsonConverterFactory.create())
        .build()

    fun getInstance(): API {
        return instance!!.create(API::class.java)
    }
}